### PR TITLE
Added Close functionality to the search modal.

### PR DESCRIPTION
**Pull request by robertwollny-pnmac**
Added Close functionality to the search modal X icon. Now when you click the X icon on the search modal it will close and not make an api request.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Sweet! definitely something thats needed at the moment.

My big concern is why we are handling this through redux. I see you are mapping it to the store with the value `searchResult.handleSearchClose`, but I don't see it in initial state, nor do I see a reducer for handling that action. Maybe we just handle closing the modal within the component?

File: src/Components/Scenes/Chrome/Search/Search-Container.js [hunk 1]
```diff
@@ -3,21 +3,24 @@ import Search from './Search';
...
+  handleKeyPress,
+  handleSearchClose
 } from '../../../../Actions/searchResultActions';
 
 const mapStateToProps = ({ searchResult }) => {
   return {
     searchInput: searchResult.searchInput,
-    isSearchClicked: searchResult.isSearchClicked
+    isSearchClicked: searchResult.isSearchClicked,
+    closeModal: searchResult.handleSearchClose
```
Comment (SelcukEdma) [inline src/Components/Scenes/Chrome/Search/Search-Container.js]: I don think this should be here. You are not using closeModal in component.
and there is not handleSearchClose in initialState

**Review by SelcukEdma — COMMENTED** [review]
Sweet! Just small things needs to remove.

**Review by ghost — COMMENTED** [review]
Per our conversation, I am aware of the redux side of things. I think you are able to remove the `closeModal` property in `mapStateToProps`. Dont think its used anywhere in your component.

File: src/Components/Scenes/Chrome/Search/Search-Container.js (hunk 1 above)
Comment (robertwollny-pnmac) [reply src/Components/Scenes/Chrome/Search/Search-Container.js]: Good catch I removed that from mapStateToProps.

**Review by ghost — APPROVED** [review]

**Review by SelcukEdma — APPROVED** [review]

**Pull request merged**